README.md

```diff
@@ -1,32 +1,70 @@
 # Digitising Medieval Irish Genealogies in RDF
 
-Traditional Irish genealogies represented as Turtle RDF graphs.
-
-This project is to transform the early Irish geneologies from HTML and
-XML in the [CELT](http://celt.ucc.ie) site into a computer readable
-[Turtle RDF](http://www.w3.org/TR/turtle/).  This will allow
-researchers to manipulate and visualize the genealogies
-programmatically.  This should allow researchers a more reliable and
-quicker way to study the various genealogies.
-
-# Conventions
-
-There are a couple of conventions used to make referencing the various
-people in the graph easier.  First, when a name is repeated, it's IRI
-is appended with a random identifier of the first eight characters
-created by uuidgen.  Second, when there are differing variants of a
-name the OWL sameAs property is used.  Third, if there are two or more
-variants of a particular decendant line, the same technique is use
-where a new node with a uuidgen id is created but is connected to the
-original node by an owl sameAs property.  The alternate line is then
-attached to the uuidgen node.
-
-Each set of files are listed by the manuscript they derive from. For
-instance, files in the LL directory come from the Book of Leinster.
-LL comes from the standard scholarly abbreviation for the book.
-
-For the moment, until there is a proper IRI to reference, all the
-turtle files use http://example.com as their base IRI. This makes
-namespacing and linking easier over the files.  For instance,
-dáil_caiss.ttl is in the LL directory and has a base IRI of
-http://example.com/LL/dáil_caiss.ttl.
\ No newline at end of file
+The goal of this project is to create a database of the early Irish
+Genealogies.  To achieve this goal and due to the nature of the source
+material, the curators chose the [Resource Description
+Framework](https://www.w3.org/TR/rdf11-primer/) (RDF) to represent it.
+Because this is a human curated database, a human readable
+representation of RDF was needed, which, in this case, the curators
+chose the [Turtle](https://www.w3.org/TR/turtle/) concrete
+representation.
+
+# Structure
+
+The database is structured by dividing the genealogies by manuscript.
+Each manuscript is given its own directory which is derived from its
+common scholarly abbreviation.  For instance, all genealogies that
+are derived from the _Book of Leinster_ are placed in the LL
+directory. As for the ontologies, these are placed in the top
+directory.
+
+## Items
+
+Each genealogy is divided into its "items" which represent one Turtle
+file in its directory.  The item file name is created from its
+manuscript header.  For instance "Aisneidem Di Araill" from the _Book
+of Leinster_ has the file name <pre>aisneidem_di_araill.ttl</pre>.
+
```

Comment:
Do we need to warn the reader at this point that we have not been consistent with regards to including or dropping "Genelogia..." / "De Genelach ..." etc. from item titles? I think we've tended to include it in Rawl. B.502 and omit it in LL.